### PR TITLE
add 2024 changes

### PR DESCRIPTION
**Pull request by ramjayakumar21**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Resolves # (issue)

## Type of change

What types of changes does your code introduce to this project?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

- [ ] Test A
- [ ] Test B

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. This is simply a reminder of what we are going to look for before merging your code._

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: content/english/pages/active-aircraft.md [hunk 1]
```diff
@@ -9,28 +9,28 @@ draft: false
...
+<--->
 
-{{% /markdown %}}
+{{% markdown class="text-center" %}}
 
-<--->
+## Albatross
 
-{{< image src="images/aircraft/dragonfly.jpg" caption="" alt="Our newest vertical take-off and landing (VTOL) drone" height="" width="600" position="center" command="Fit" class="" title="" caption="Dragonfly in its early stages without wings, a tail, and a forward motor." >}}
+The Albatross was the team's premier aircraft for the the 2023-2024 season. Built off the base of the Dragonfly, the Albatross is a VTOL aircraft that is designed to be super modular, allowing for quick and easy payload changes as well as easy disassembly. The team is currently working on the Albatross to improve its flight performance and capabilities.
```
Comment (ElioDiNino) [inline content/english/pages/active-aircraft.md]:
```suggestion
Albatross was the team's premier aircraft for the 2023-2024 season. Built based on Dragonfly, Albatross is a VTOL aircraft that is designed to be super modular, allowing for quick and easy payload changes as well as easy disassembly. The team is currently working on Albatross to improve its flight performance and capabilities.
```

File: content/english/pages/active-aircraft.md [hunk 2]
```diff
@@ -9,28 +9,28 @@ draft: false
...
+
+The Albatross performed in the 2024 AEAC and 2024 SUAS competitions, where it was able to place 3rd overall at AEAC!
```
Comment (ElioDiNino) [inline content/english/pages/active-aircraft.md]:
```suggestion
Albatross competed in the 2024 AEAC and SUAS competitions. It was able to place 3rd overall at AEAC!
```

File: assets/images/competition/20240524_104844.jpg
Comment (ElioDiNino) [inline assets/images/competition/20240524_104844.jpg:1]: File name

File: assets/images/competition/20240626_123417.jpg
Comment (ElioDiNino) [inline assets/images/competition/20240626_123417.jpg:1]: File name

File: assets/images/competition/IMG_2504.png
Comment (ElioDiNino) [inline assets/images/competition/IMG_2504.png:1]: File name

File: assets/images/competition/UBC.jpg
Comment (ElioDiNino) [inline assets/images/competition/UBC.jpg:1]: Bad photo, should not go on the website

File: assets/images/faq/5D3_8793.JPG
Comment (ElioDiNino) [inline assets/images/faq/5D3_8793.JPG:1]: Not a great image imo

File: assets/images/faq/1.jpg
Comment (ElioDiNino) [inline assets/images/faq/1.jpg:1]: We don't need to remove unnecessarily, the more the merrier

File: assets/images/faq/Screenshot 2024-08-17 at 11.38.57 AM.png
Comment (ElioDiNino) [inline assets/images/faq/Screenshot 2024-08-17 at 11.38.57 AM.png:1]: Meh photo

File: assets/images/faq/Screenshot 2024-08-17 at 11.40.03 AM.png
Comment (ElioDiNino) [inline assets/images/faq/Screenshot 2024-08-17 at 11.40.03 AM.png:1]: File name

File: assets/images/old/competition/1.jpg
Comment (ElioDiNino) [inline assets/images/old/competition/1.jpg:1]: Keep this, just put it after new ones added

File: assets/images/old/competition/2.jpg
Comment (ElioDiNino) [inline assets/images/old/competition/2.jpg:1]: Keep this, just put it after new ones added

File: assets/images/old/competition/3.jpg
Comment (ElioDiNino) [inline assets/images/old/competition/3.jpg:1]: Keep this, just put it after new ones added

File: assets/images/old/competition/4.jpg
Comment (ElioDiNino) [inline assets/images/old/competition/4.jpg:1]: Keep this, just put it after new ones added

File: assets/images/people/team_photo2.jpg
Comment (ElioDiNino) [inline assets/images/people/team_photo2.jpg:1]: Why do we need this variant?

File: content/english/pages/active-aircraft.md [hunk 3]
```diff
@@ -40,4 +40,11 @@ The Hummingbird Project was the team’s response to the need for a more precise
 As the years have progressed, Hummingbird has retained its strong performance and so it continues to be used as a test drone for software and camera systems.
 
 {{% /markdown %}}
-{{< /columns >}}
\ No newline at end of file
+
+<--->
+
+{{< image src="images/aircraft/hummingbird-1.jpg" caption="" alt="closeup photo of the Hummingbird sitting in the grass" height="" width="600" position="center" command="Fit" class="" title="" caption="" >}}
```
Comment (ElioDiNino) [inline content/english/pages/active-aircraft.md]:
```suggestion
{{< image src="images/aircraft/hummingbird-1.jpg" caption="" alt="A close-up photo of Hummingbird sitting in the grass" height="" width="600" position="center" command="Fit" class="" title="" caption="" >}}
```

File: content/english/pages/competitions.md [hunk 4]
```diff
@@ -19,7 +19,7 @@ draft: false
...
+For the 2024 AEAC Student UAS Competition, the team undertook two challenging tasks. In Task 1, the team faced unexpected difficulties due to strong winds and an imbalanced aircraft, which prevented them from completing the mission as planned. However, they made critical modifications to their aircraft for Task 2, including removing the tail and wings and optimizing the center of gravity, which resulted in a successful and stable flight. Despite some technical challenges, such as GPS inaccuracies, the team managed to perform a precise manual landing. Overall, the team performed well, achehiving 3rd place while providing valuable learning experiences, and the team is eager to apply these insights to future projects.
```
Comment (ElioDiNino) [inline content/english/pages/competitions.md]: This text is supposed to be about the competition, not about how we performed. The tasks details are to give an example of what can be expected

File: content/english/pages/competitions.md [hunk 5]
```diff
@@ -35,7 +35,7 @@ For the 2022 competition, AEAC created a simulated embassy protection mission. O
...
 
-UBC UAS built a large, fast, multi-rotor capable of performing autonomous flights while carrying multiple payloads for the SUAS competition. The mission was to autonomously fly to waypoints while avoiding obstacles like buildings. We were also tasked with taking aerial photos to stich together to create a map, creating an active aircraft avoidance system, and building a rover to be deployed that autonomously navigates and transports a water bottle to the mission location. Our in-house software suite was used to create aircraft flight paths, manage imaging for mapping, sending telemetry to the control server and perform object detection. The competition ran from July 15th to 18th.
+For the SUAS 2024 competition, the main competition involved following a loop and then completing a payload drop. Despite  preparation, the intense heat at the competition site presented unexpected challenges, affecting the aircraft's performance and the accuracy of our payload drop. The aircraft did take off but was unable to complete the route as required. However, the experience was still unforgettable and the team looks forward to refining our techniques and returning stronger next year.
```
Comment (ElioDiNino) [inline content/english/pages/competitions.md]: Same comment as above

File: content/english/pages/legacy-aircraft.md [hunk 6]
```diff
@@ -8,6 +8,28 @@ description: "Past aircraft build and flown by the team."
 draft: false
 ---
 
+
+{{< columns >}}
+
+
+{{< image src="images/aircraft/dragonfly.jpg" caption="" alt="Our newest vertical take-off and landing (VTOL) drone" height="" width="600" position="cen ter" command="Fit" class="" title="" caption="Dragonfly in its early stages without wings, a tail, and a forward motor." >}}
```
Comment (ElioDiNino) [inline content/english/pages/legacy-aircraft.md]:
```suggestion
{{< image src="images/aircraft/dragonfly.jpg" caption="" alt="Our newest vertical take-off and landing (VTOL) drone" height="" width="600" position="center" command="Fit" class="" title="" caption="Dragonfly in its early stages without wings, a tail, and a forward motor." >}}
```

File: content/english/pages/legacy-aircraft.md [hunk 7]
```diff
@@ -8,6 +8,28 @@ description: "Past aircraft build and flown by the team."
...
+
+
+<--->
+
+{{% markdown class="text-center" %}}
+
+## Dragonfly
+
+
+The team's second VTOL project to date, the Dragonfly was built to be a modular drone that allowed for easy upgrades and payload swapping. The team used the knowledge gained from this aircraft to build the Albatross after it's development concluded in the summer of 2023.
```
Comment (ElioDiNino) [inline content/english/pages/legacy-aircraft.md]:
```suggestion
The team's second VTOL project to date, Dragonfly was built to be a modular drone that allowed for easy upgrades and payload swapping. The team used the knowledge gained from this aircraft to build Albatross after its development concluded in the summer of 2023.
```

File: hugo.toml [hunk 8]
```diff
@@ -14,7 +14,7 @@ summaryLength = 10 # see https://gohugo.io/content-management/summaries/
 # google analytics
 googleAnalytics = "G-MEASUREMENT_ID" # see https://gohugo.io/templates/internal/#configure-google-analytics
 # disqus short name
-disqusShortname = "" # we use disqus to show comments in blog posts. To install disqus please follow this tutorial https://portfolio.peter-baumgartner.net/2017/09/10/how-to-install-disqus-on-hugo/
+disqus.shortname = "" # we use disqus to show comments in blog posts. To install disqus please follow this tutorial https://portfolio.peter-baumgartner.net/2017/09/10/how-to-install-disqus-on-hugo/
```
Comment (ElioDiNino) [inline hugo.toml:17]: Was this a manual change or from a deps update?

File: assets/images/aircraft/albatross-1.jpeg
Comment (ElioDiNino) [inline assets/images/aircraft/albatross-1.jpeg:1]: File name

File: content/english/pages/recruitment.md [hunk 9]
```diff
@@ -8,22 +8,219 @@ description: "Find out all the details about our recruitment process!"
...
+## Position Openings
+{{% /markdown %}}
+
+{{< toc >}}
+
+{{% markdown class="text-center pt-4 pb-3" %}}
+## General Member Roles
+{{% /markdown %}}
+
+UBC UAS has three subteams for general members: **Aircraft**, **Payload**, and **Software**. Each subteam has divisions that focus on specific skillsets within the subteam.
```
Comment (ElioDiNino) [inline content/english/pages/recruitment.md]:
```suggestion
UBC UAS has three sub-teams for general members: **Aircraft**, **Payload**, and **Software**. Each sub-team has divisions that focus on specific skillsets within the sub-team.
```

File: content/english/pages/recruitment.md [hunk 10]
```diff
@@ -8,22 +8,219 @@ description: "Find out all the details about our recruitment process!"
...
+
+{{% markdown class="text-center pt-4 pb-3" %}}
+### Aircraft Roles
+
+The aircraft subteam builds innovative uncrewed aerial vehicles for our competitions. We work closely with our software and payload subteams to ensure proper integration between our systems. We are also responsible for the upkeep of active aircraft and ensuring flight readiness for test flights and competition.
```
Comment (ElioDiNino) [inline content/english/pages/recruitment.md]:
```suggestion
The aircraft sub-team builds innovative uncrewed aerial vehicles for our competitions. It works closely with our software and payload sub-teams to ensure proper integration between our systems. The aircraft sub-team is also responsible for the upkeep of active aircraft and ensuring flight readiness for test flights and competition.
```

File: content/english/pages/recruitment.md [hunk 11]
```diff
@@ -8,22 +8,219 @@ description: "Find out all the details about our recruitment process!"
...
+Design and manufacture aerodynamic and lightweight airframes as well as control surfaces and landing gear. Members will learn and apply skills in mechanical design, aerodynamics, manufacturing, control surface selection, airfoil analysis and simulation tools applicable across industries.
+
+#### ELEC Aircraft
+
+Design of electrical systems within our aircraft, working closely with software and payload to ensure proper integration between systems. Members will learn and apply skills that are essential across industries including communication systems management, power management, signal processing, soldering, PCB design, and PID tuning and sensor calibration.
+
+{{% markdown class="text-center pt-4 pb-3" %}}
+### Payload Roles
+
+The payload subteam creates mission-specific payload systems for competition. The subteam works on projects including autonomous delivery rovers, winch mechanisms, and anything capable of interfacing with the aircraft. These projects involve the integration of multiple engineering disciplines.
```
Comment (ElioDiNino) [inline content/english/pages/recruitment.md]:
```suggestion
The payload sub-team creates mission-specific payload systems for competition. The sub-team works on projects including autonomous delivery rovers, winch mechanisms, and anything capable of interfacing with the aircraft. These projects involve the integration of multiple engineering disciplines.
```

File: content/english/pages/recruitment.md [hunk 12]
```diff
@@ -8,22 +8,219 @@ description: "Find out all the details about our recruitment process!"
...
+Work alongside electrical designers to develop enclosures and structures controlled by electrical components. You will make use of CAD, FEA software, and various manufacturing techniques including 3D printing and laser cutting. Knowledge of statics, materials, and mechanical design principles is an asset.
+
+#### ELEC Payload
+
+Design segments of payload projects with electrical aspects. You will learn to use microcontrollers to control motors and circuits, using EDA, RC, power distribution, etc. to ensure proper control and movements of the payload components.
+
+{{% markdown class="text-center pt-4 pb-3" %}}
+### Software Roles
+
+The software team assigns roles based on preference and need, where members works on many different projects/programs and uses a variety of tools and languages. We encourage members to work on projects that they are interested in and apply their experience while learning new tools.
```
Comment (ElioDiNino) [inline content/english/pages/recruitment.md]:
```suggestion
The software team assigns roles based on preference and need, where members works on many different projects/programs and uses a variety of tools and languages. Members are encouraged to work on projects that they are interested in and apply their experience while learning new tools.
```

File: content/english/pages/recruitment.md [hunk 13]
```diff
@@ -8,22 +8,219 @@ description: "Find out all the details about our recruitment process!"
...
+
+
+{{% /markdown %}}
+
+#### Software Member
+All projects are up for grabs (everyone gets to choose at the start of the year what they get to work on) and we plan to work with a sprint based system. Some languages and tools we use: JavaScript, Go, C++, Git, Angular, Python, React, Django, Docker, Linux CLI. The Some developers may work on Payload subteam projects, which requires embedded development skills, including the use of microcontrollers, Git, control systems, telemetry, and more. Knowledge of motor control, control systems, and microcontrollers is an asset. Any knowledge with these tools and languages are a plus. Prior professional software development experience is not required but recommended. Open source contribution is also a plus.
```
Comment (ElioDiNino) [inline content/english/pages/recruitment.md]:
```suggestion
All projects are up for grabs (everyone gets to choose at the start of the year what they get to work on) and the team works on a sprint-based system. Some languages and tools the team uses: JavaScript, Go, C++, Git, Angular, Python, React, Django, Docker, and Linux. Some developers may work on Payload sub-team projects, which require embedded development skills, including the use of microcontrollers, Git, control systems, telemetry, and more. Knowledge of motor control, control systems, and microcontrollers is an asset. Any knowledge with these tools and languages is a plus. Prior professional software development experience is not required but strongly recommended. Open-source contributions are also a plus.
```

File: content/english/pages/recruitment.md [hunk 14]
```diff
@@ -8,22 +8,219 @@ description: "Find out all the details about our recruitment process!"
...
+### Admin Team Roles
+{{% /markdown %}}
+
+#### Events Coordinator
+
+The events coordinator is responsible for organizing and planning team events. This includes team socials, workshops, and trips. The events coordinator will also be responsible for organizing and planning public events such as recruitment events. This role is a great opportunity to develop your leadership, communication, and organizational skills.
+
+#### Finance Manager
+
+The finance manager is responsible for managing the team’s finances. This includes managing the team’s budget, applying for grants, managing sponsorships, and delegating finance work to executive members. This role is a great opportunity to develop your financial knowledge, leadership, and organizational skills.
```
Comment (ElioDiNino) [inline content/english/pages/recruitment.md]:
```suggestion
The finance manager is responsible for managing the team’s finances. This includes managing the team’s budget, applying for grants, processing purchases, and delegating finance work to executive members. This role is a great opportunity to develop your financial knowledge, leadership, and organizational skills.
```

File: content/english/pages/recruitment.md [hunk 15]
```diff
@@ -8,22 +8,219 @@ description: "Find out all the details about our recruitment process!"
...
+{{< accordion title="Where do I apply?" >}}
+This webpage (ubcuas.com/recruitment)! All the application details and links will be updated on August 25th.
+{{< /accordion >}}
+
+{{< accordion title="What do I need to apply?" >}}
+The application process includes a general submission (with your resume, desired position, as well as some other information that will be released on August 25th). Be ready to show your eagerness and passion!
+{{< /accordion >}}
+
+{{< accordion title="What are the different roles?" >}}
+There are openings for our 3 subteams: Aircraft, Payload, and Software. Within the Aircraft and Payload teams, we have divisions for ELEC and MECH, with tasks within those divisions being related to their discipline:
```
Comment (ElioDiNino) [inline content/english/pages/recruitment.md]:
```suggestion
There are openings for our 3 sub-teams: Aircraft, Payload, and Software. Within the Aircraft and Payload teams, we have divisions for ELEC and MECH, with tasks within those divisions being related to their discipline:
```

**Review by ElioDiNino — DISMISSED** [review]
Please see review comments

File: hugo.toml (hunk 8 above)
Comment (ramjayakumar21) [reply hugo.toml:17]: Had to change it to run the application

**Comment by CharlesSurianto** [comment]
lgtm

**Review by CharlesSurianto — APPROVED** [review]

**Pull request merged**